doc/getting-started.md

```diff
@@ -6,14 +6,7 @@ The most convenient way to interact with docat is with it's official CLI tool, [
 
 You can download a standalone binary of the latest release for your platform [here](https://github.com/docat-org/docatl/releases/latest) or [use go](https://github.com/docat-org/docatl#using-go) or [docker](https://github.com/docat-org/docatl#using-docker) to install it.
 
-After you've installed `docatl`, just point it to your documentation folder, which will be packaged and pushed to `docat`.
-To do this, use the following command:
-
-```sh
-docatl push --host http://localhost:8000 ./docs/ awesome-project 1.0.0
-```
-
-Use `docatl --help` to discover all commands available to manage your `docat` documentation!
+The `docatl` alternatives to the `curl` commands are outlined below. Do note that the host address and api-key can be omitted if specified in a `.docatl.yml` file. See the `docatl` documentation for more information.
```

Comment:
I would still keep that here 
```
Use `docatl --help` to discover all commands available to manage your `docat` documentation!
```

> See the `docatl` documentation for more information.

should contain a link to some docs

> The `docatl` alternatives to the `curl` commands are outlined below.

not very happy with this sentence but failed to come up with a better structure